packages/intentalyzer/src/entities.ts

```diff
@@ -4,6 +4,7 @@ export interface Entity {
     readonly utteranceOffsets?: {
         startIndex: number;
         endIndex: number;
+        length?: number;
```

Comment:
Trying to figure out how this got in here...? I remember doing some work right after M2 to add this and I honestly forgot where it left off, but I'm wondering why it's showing up here in this PR now?

Reply:
Honestly, I don't remember. I just needed to add this as an optional field for the project to compile

P.S.: sorry for the delayed response, I've been moving this week so my life is in boxes

Reply:
It seems redundant, since it can be calculated from the difference of `startIndex` and `endIndex`. The compile time error is shown in the PR description